jwst/cube_build/ifu_cube.py

```diff
@@ -284,6 +297,13 @@ def set_geometry(self, corner_a, corner_b, lambda_min, lambda_max):
         na = math.ceil(xilimit / self.cdelt1) + 1
         nb = math.ceil(etalimit / self.cdelt2) + 1
 
+        # if the user set the nspax_xi or nspax_eta then redefine na, nb
+        # it is assumed that both values are ODD numbers
```

Comment:
So, given the confusion above, do you really mean ODD here or not?

Reply:
My bad. I had not cleaned that up yet. I originally thought the numbers needed to be even until I tested it and realized I needed them to odd.